prep-javascript-conditionals/main.js

```diff
@@ -0,0 +1,59 @@
+const age = 24;
+function isAdult(age) {
+  if (age < 18) {
+    return 'false';
```

Comment:
make sure your `true` and `false` values don't have quotes around them, that make them strings.